### PR TITLE
[FIX] Add missing labels 4118

### PR DESCRIPTION
**Pull request by hschink**
Fixes [ticket 4118](https://www.openproject.org/work_packages/4118)

### CODE REVIEW

**Comment by mugwump** [comment]
&nbsp;<b>Code Climate</b> has <a href="https://codeclimate.com/repos/528f35c8c7f3a33b7d03cde8/compare/fix/add_missing_labels_4118">analyzed this pull request</a>.

File: app/views/projects/settings/_activities.html.erb [hunk 1]
```diff
@@ -43,7 +43,7 @@ See doc/COPYRIGHT.rdoc for more details.
   <%= fields_for "enumerations[#{enumeration.id}]", enumeration do |ff| %>
   <tr class="<%= cycle('odd', 'even') %>">
     <td align="center">
-      <%= label_tag "enumerations_#{enumeration.id}_active", l(:description_active), :class => "hidden-for-sighted" %>
+      <%= label_tag "enumerations_#{enumeration.id}_active", "#{h(enumeration)} #{l(:description_active)}", :class => "hidden-for-sighted" %>
```
Comment (sschu) [inline app/views/projects/settings/_activities.html.erb]: I am a little curious - why can you just write "h(enumeration)"? I would assume enumeration is a model and you would need something like "enumeration.name" ...

Comment (ulferts) [inline app/views/projects/settings/_activities.html.erb]: enumeration.rb: 128

<pre>
  def to_s; name end
</pre>

Comment (sschu) [inline app/views/projects/settings/_activities.html.erb]: Magic...

Comment (ulferts) [inline app/views/projects/settings/_activities.html.erb]: `enumeration.to_s` would still be clearer

**Pull request merged**